test/negotiation/utils/NegotiationTest.js

```diff
@@ -0,0 +1,473 @@
+const puppeteer = require('puppeteer-core');
+const cliProgress = require('cli-progress');
+const axios = require('axios');
+const expect = require('chai').expect;
+
+const ClientStream = require('./ClientStream');
+const ClientConnection = require('./ClientConnection');
+const ErizoConnection = require('./ErizoConnection');
+const SdpChecker = require('./SdpUtils');
+
+const path = 'http://omahaproxy.appspot.com/all.json';
+const platform = 'mac';
+const chromeVersion = 'canary';
+
+function sleep(ms) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
+
+let browserInstalled = false;
+let revisionInfo;
+
+const installBrowser = async () => {
```

Comment:
I'm not sure about having this here. I would probably separate it and have it be a prerequisite for running the tests.  That script could, for instance, take environment variables as input for the chromeVersion and/or the platform

Reply:
I moved this code to a new `BrowserInstaller.js` file